### PR TITLE
Update doc and fix example with new tagging info

### PR DESCRIPTION
**Pull request by siddharthgoel88**

### CODE REVIEW

File: examples/creating-interface-endpoint-to-endpoint-service/main.tf [hunk 1]
```diff
@@ -6,6 +6,10 @@ provider "aws" {
 # Local values declaration. Change the value with a valid value before you used this example
 #############################################################################################
 locals {
+  product_domain = "tsi"
+  service_name   = "tsi-service"
+  enviroment     = "testing"
```
Comment (febryantonius) [inline examples/creating-interface-endpoint-to-endpoint-service/main.tf]: typo
```suggestion
  environment     = "testing"
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ Usage
 ```hcl
 module "service-consumer-privatelink" {
   source                = "github.com/traveloka/terraform-aws-privatelink-consumer?ref=master"
+
+  product_domain = "tsi"
+  service_name   = "tsi-service"
```
Comment (febryantonius) [inline README.md]: to inform the user that this resource is owned by them but the service name should be their dependency service name
```suggestion
  service_name   = "beicalc"
```

File: examples/creating-interface-endpoint-to-endpoint-service/main.tf [hunk 3]
```diff
@@ -6,6 +6,10 @@ provider "aws" {
 # Local values declaration. Change the value with a valid value before you used this example
 #############################################################################################
 locals {
+  product_domain = "tsi"
+  service_name   = "tsi-service"
```
Comment (febryantonius) [inline examples/creating-interface-endpoint-to-endpoint-service/main.tf]:
```suggestion
  service_name   = "beicalc"
```

File: examples/creating-interface-endpoint-to-endpoint-service/main.tf [hunk 4]
```diff
@@ -6,6 +6,10 @@ provider "aws" {
 # Local values declaration. Change the value with a valid value before you used this example
 #############################################################################################
 locals {
+  product_domain = "tsi"
+  service_name   = "beicalc"
+  environment     = "testing"
```
Comment (rafikurnia) [inline examples/creating-interface-endpoint-to-endpoint-service/main.tf]: looks like it is not formatted well, can you try to execute `terraform fmt` please

Comment (siddharthgoel88) [reply examples/creating-interface-endpoint-to-endpoint-service/main.tf]: Awesome catch. I like nitpicks :-) Fixed it now. Just to let you know I have run `terraform fmt` on the complete repo so there is minor formatting done in `main.tf` as well.

**Pull request merged**